composer.json

```diff
@@ -11,7 +11,7 @@
     "require": {
         "php": ">=5.4.0",
         "league/flysystem": "~1.0",
-        "aws/aws-sdk-php": "3.*@dev"
+        "aws/aws-sdk-php": "^3.0.0"
```

Comment:
No. `~3.0` is the standard being used in flysystem. It makes no sense to change it when it does the same thing in this case.